services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from datetime import time
+
+from django.db.models.query import QuerySet
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: time,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        queryset = MovieSession.objects.filter(show_time__date=session_date)
+    return queryset
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: time = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> None:
+    queryset = MovieSession.objects.get(id=session_id)

```

Comment:
uniformative name for variable - is method get returns queryset ? 
https://docs.djangoproject.com/en/3.0/topics/db/queries/#retrieving-a-single-object-with-get:~:text=If%20you%20know%20there%20is%20only%20one%20object%20that%20matches%20your%20query%2C%20you%20can%20use%20the%20get()%20method%20on%20a%20Manager%20which%20returns%20the%20object%20directly%3A

Reply:
got it, thanks!